hudi-utilities/src/main/java/org/apache/hudi/utilities/deltastreamer/DeltaSync.java

```diff
@@ -455,6 +459,12 @@ public void refreshTimeline() throws IOException {
       case BULK_INSERT:
         writeStatusRDD = writeClient.bulkInsert(records, instantTime);
         break;
+      case INSERT_OVERWRITE:
+        writeStatusRDD = writeClient.insertOverwrite(records, instantTime).getWriteStatuses();
+        break;
+      case INSERT_OVERWRITE_TABLE:
```

Comment:
Do we plan to add "Delete_partition" in a follow up PR? Not looking to expand the scope of this PR. just asking

Reply:
Haven't thought about it.
we can definitely do it.
I will do it in the following pr.

Reply:
created the following tasks for `delete_partition` support in deltaStreamer
https://issues.apache.org/jira/browse/HUDI-2189